pogobuf/pogobuf.utils.js

```diff
@@ -248,6 +248,30 @@ module.exports = {
     },
 
     /**
+     * Utility method to get the distance in meters between two locations
+     * @param {number} lat1 - origin latitude
+     * @param {number} lng1 - origin longitude
+     * @param {number} lat2 - destination latitude
+     * @param {number} lng2 - destination longitude
```

Comment:
These could be named like:

- originLatitude
- originLongitude
- destinationLatitude
- destinationLongitude

or destLng etc.